packages/frontend/src/components/RichTextEditor/StepVariablePlugin.ts

```diff
@@ -92,7 +92,7 @@ export const StepVariable = Node.create<VariableOptions>({
     ]
   },
   renderText({ node }) {
-    return node.attrs.id
+    return `{{${node.attrs.id}}}`
```

Comment:
ps @stanleynguyen can i ask when does this occur ah

Reply:
This is when we do `editor.getText()`